local-ui-dev/docker-compose.yaml

```diff
@@ -0,0 +1,65 @@
+version: "3"
+
+# This docker-compose is for developer convenience, not for running in production.
+networks:
```

Comment:
Can you just add some of these extra configuration lines, such as external network name and container name, to the root docker-compose.yaml and use that for local development? That root docker-compose is meant for development as well, and it looks like there is very little difference between this and that file.

Reply:
Refactored this to use one docker-compose; enabled by adding external docker network creating as a make task and a dependency of all the docker-compose running tasks, and an env variable SPEC_RELEASE_PATH which can override the SPEC_RELEASE_PATH in docker-compose, since the local-ui-dev workflow relies on all collections being available.